### PR TITLE
416-vsphere-ipi-transit-masq-test

### PR DESCRIPTION
**Pull request by jechen0648**

### CODE REVIEW

File: ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/ovn/custom-transit-switch-masquerade-subnet/cucushift-installer-rehearse-vsphere-ipi-ovn-custom-transit-switch-masquerade-subnet-workflow.yaml [hunk 1]
```diff
@@ -4,6 +4,7 @@ workflow:
     pre:
     - chain: cucushift-installer-rehearse-vsphere-ipi-ovn-custom-transit-switch-masquerade-subnet-provision
     post:
+    - ref: wait
```
Comment (zhaozhanqi) [inline ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/ovn/custom-transit-switch-masquerade-subnet/cucushift-installer-rehearse-vsphere-ipi-ovn-custom-transit-switch-masquerade-subnet-workflow.yaml:7]:
```suggestion
    - ref: wait](ref: cucushift-hypershift-extended-debug)
```

**Comment by jechen0648** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-vsphere-ipi-transit-masq-subnet-f28

**Comment by jechen0648** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-vsphere-ipi-transit-masq-subnet-f28

**Comment by jechen0648** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-vsphere-ipi-transit-masq-subnet-f28

**Pull request closed without merging**